### DISCUSSION TITLE
How about % for jumping between #+BEGIN_<block> and #+END_<block>?

### DISCUSSION THREAD

**Discussion by Kazark** [syl20bnr/spacemacs#14738] [Ideas]
[Relevant](https://www.reddit.com/r/emacs/comments/ag5cxz/how_to_jump_to_beginend_of_any_orgblock/).

The jump function is probably even something I could figure out how to implement if I did it raw; however I suspect that `%` is already bound in Evil and so I would need to rather define some kind of text object or something---comments appreciated.

**Comment by dalanicolai** [comment]
Are you asking, for opinions about this jump functionality? Or about information how to use text object commands with evil mode?

For the latter you can find evil interactive codes [here](https://github.com/noctuid/evil-guide#interactive-codes)
or in `evil-types.el` (in that file look for `evil-define-interactive-code`).

Also, I did not read [this](https://www.reddit.com/r/emacs/comments/ag5cxz/how_to_jump_to_beginend_of_any_orgblock/), but it might be helpful (e.g. I see it mentions org-elements which might provide your "text-objects")
